guides/installing_esphome.rst

```diff
@@ -0,0 +1,97 @@
+Installing ESPHome
```

Comment:
Maybe title this "Manually installing ESPHome" or something like that?

Reply:
"Getting Started with the ESPHome Command Line"